### ISSUE TITLE
File does not open new window on first try and window is not on top

### ISSUE THREAD

**Issue by LasseBoerresen** [Codealike/Codealike-Atom#16]
I isolated codealike plugin for atom as the source of the two issues I was having with atom, by method of exclusion. I would have to open each file twice in order for them to open properly in an new atom window. Also, that window was not automatically on top. When I disabled the plugin, the issues were gone.

**Comment by soke1556** [comment]
Thanks @LasseBoerresen cannot reproduce this. Can you enumerate the steps you're following? Thanks.

**Comment by LasseBoerresen** [comment]
I don't really know how to be much more specific. I installed atom on 2018-01-22 on windows 10 64-bit, from AtomSetup-x64.exe downloaded from the https://atom.io/. I immediately installed several plugins, including minimap, linter and codealike. I can't remember when I noticed the buggy behavior. A month later, I selectively tried disabling plugins to see if any was the root of the behavior and codealike was the trigger.

**Comment by soke1556** [comment]
@LasseBoerresen thanks. 

What you're doing is this (see screenshot) and then it simply doesn't open the file on the first try?

![image](https://user-images.githubusercontent.com/1424405/36788531-0c399322-1c6d-11e8-8904-ad2ac7b83271.png)

**Comment by LasseBoerresen** [comment]
No, I open a file in windows explorer or winSCP and I just get this screen, see screenshot. I did a bit more testing, and it seems the problem only exists the first time I open a file in a new directory (atom project). This also explains my experience of it happening more frequently in winSCP, because it creates a temp directory for each session. 

Lasse

![image](https://user-images.githubusercontent.com/2159782/36794223-093f0b92-1ca0-11e8-83f8-58851f480e34.png)

**Comment by soke1556** [comment]
Thanks @LasseBoerresen now I was able to reproduce it.

@diglesias17 This is related to codealike.json. If codealike.json is not present in the folder, then the file is not show in Atom when it's open from Windows Explorer. If codealike.json is present, then it opens at once. Make sure the file is always open even when codealike.json is not present and is being created.
